operator/v1/types_console.go

```diff
@@ -143,8 +143,97 @@ type Capability struct {
 	Visibility CapabilityVisibility `json:"visibility"`
 }
 
```

Comment:
@opayne1 Thank you for the feedback.  This file and `operator/v1/tests/consoles.operator.openshift.io/AAA_ungated.yaml` are the only files containing docs written. The other files are auto-generated from this file, so add your feedback to this file if you find any other content that needs updating.